### PR TITLE
qmc2 : init at 0.195

### PR DESCRIPTION
**Pull request by bignaux**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by grahamc** [comment]
@GrahamcOfBorg eval

File: pkgs/misc/emulators/qmc2/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  name = "qmc2-${version}";
+  version = "0.195";
+
+  src = fetchurl {
+      url = "https://netix.dl.sourceforge.net/project/qmc2/qmc2/${version}/${name}.tar.gz";
+      #url = "mirror://sourceforge/projects/qmc2/files/qmc2/${version}/${name}.tar.gz/download";
```
Comment (grahamc) [inline pkgs/misc/emulators/qmc2/default.nix]: Please delete this comment, anstead of using this `netix...` url use the mirror we have, like this: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/misc/mantisbt.nix#L13

File: pkgs/misc/emulators/qmc2/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{ stdenv
+, fetchurl, qmake, qttools, pkgconfig
+, minizip, zlib
+, qtbase, qtsvg, qtmultimedia, qtwebkit, qttranslations, qtxmlpatterns
+, rsync, SDL2, xwininfo
+, utillinux #colrm
```
Comment (grahamc) [inline pkgs/misc/emulators/qmc2/default.nix]: Please delete the colrm comment

File: pkgs/misc/emulators/qmc2/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      sha256 = "1dzmjlfk8pdspns6zg1jmd5fqzg8igd4q38cz4a1vf39lx74svns";
+  };
+
+  nativeBuildInputs = [ qttools pkgconfig ];
+  buildInputs = [ minizip qtbase qtsvg qtmultimedia qtwebkit
+                  qttranslations qtxmlpatterns rsync SDL2
+                  xwininfo zlib utillinux xorg.libxcb ];
+
+  makeFlags = [ #"DEFINES-=QMC2_BUNDLED_MINIZIP"
+                 #"DEFINES-=QMC2_BUNDLED_ZLIB"
```
Comment (grahamc) [inline pkgs/misc/emulators/qmc2/default.nix]: Please delete these commented lines

File: pkgs/misc/emulators/qmc2/default.nix [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                 "DESTDIR=$(out)"
+                 "PREFIX=/"
+                 "DATADIR=/share/"
+                 "SYSCONFDIR=/etc" ];
+
+  meta = with stdenv.lib; {
+    description = "A Qt frontend for MAME/MESS";
+    homepage = http://qmc2.batcom-it.net;
```
Comment (grahamc) [inline pkgs/misc/emulators/qmc2/default.nix]: This URL supports HTTPS, so please switch to that

**Comment by grahamc** [comment]
lgtm other than these comments

File: pkgs/misc/emulators/qmc2/default.nix (hunk 1 above)
Comment (bignaux) [reply pkgs/misc/emulators/qmc2/default.nix]: problem is that mirror is broken for me, never works and i get their custom 404 ... boring SF.

**Comment by bignaux** [comment]
Thanks for review @grahamc .

**Comment by grahamc** [comment]
@GrahamcOfBorg build qmc2

**Comment by bignaux** [comment]
@GrahamcOfBorg build qmc2

**Comment by grahamc** [comment]
@GrahamcOfBorg build qmc2

**Pull request merged**